src/OrchardCore.Modules/OrchardCore.Users/Models/PasswordSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.Users.Models
+{
+    public class PasswordSettings
+    {
+        public bool EnableLostPassword { get; set; }
```

Comment:
`AllowResetPassword`

Reply:
I understand it will impact a lot of changes in the views too.

Reply:
Ok, EnableLostPassword => AllowResetPassword